src/random.cr

```diff
@@ -52,6 +52,32 @@ require "random/pcg32"
 module Random
   DEFAULT = PCG32.new
```

Comment:
Should we deprecate the old const in favor of this new method?

Reply:
I actually implemented it here. We might extract it before merge.